src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Semantic/Services/SemanticTokensEditsDiffer.cs

```diff
@@ -53,10 +53,10 @@ protected override bool ContentEquals(int oldTextIndex, int newTextIndex)
 
         private Container<SemanticTokensEdit> ProcessEdits(IReadOnlyList<DiffEdit> diffs)
         {
-            var results = new List<SemanticTokensEdit>();
+            var razorResults = new List<RazorSemanticTokensEdit>();
             foreach (var diff in diffs)
             {
-                var current = results.Count > 0 ? results[results.Count - 1] : null;
+                var current = razorResults.Count > 0 ? razorResults[razorResults.Count - 1] : null;
```

Comment:
Maybe:

```suggestion
                var current = razorResults.LastOrDefault();
```

Reply:
That's probably cleaner, but it means allocating an enumerator, and given the allocation crimes I'm resolving here I'm trying to be thoughtful.

Frankly I'm wondering if I ought to remove the `.Select` call on line 101 and replace it with a `foreach... yield return` just so the temptation of already having `LINQ` imported is removed.